### PR TITLE
support change $rvm_path/user/db

### PR DESCRIPTION
**Pull request by vkill**
Hi

This is a updated version about #57 

Please review it, thanks.

### CODE REVIEW

**Comment by lpaulmp** [comment]
@vkill Sorry for delay to check, I will do it today.

**Comment by vkill** [comment]
@lpaulmp thanks :)

File: tasks/rvm.yml [hunk 1]
```diff
@@ -53,3 +53,16 @@
   when: not rvm_binary.stat.exists
   become: yes
   become_user: '{{ rvm1_user }}'
+
+- name: Change rvm user/db
+  lineinfile: "dest={{ rvm1_install_path }}/user/db state=present regexp='^{{ item }}=' line='{{ item }}={{ rvm1_configurations[item] }}'"
+  with_items: rvm1_configurations.keys()
```
Comment (lpaulmp) [inline tasks/rvm.yml]: Looks here we are missing conditional like `when: rvm1_configurations.keys() is undefined` or  `when: rvm1_configurations.keys()`

**Comment by lpaulmp** [comment]
@vkill can you address the comment ?

**Comment by lpaulmp** [comment]
@vkill Feel free to re-open this issue, address the comment and rebase the branch.

**Pull request closed without merging**